scss/_buttons.scss

```diff
@@ -232,23 +232,28 @@
 
 // Boosted mod: social button
 .btn-social {
-  border-color: currentcolor;
+  --#{$prefix}btn-border-color: currentcolor;
+  --#{$prefix}btn-hover-color: #{$white};
+  --#{$prefix}btn-hover-bg: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-hover-border-color: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-active-color: #{$white};
+  --#{$prefix}btn-active-bg: #{$black};
+  --#{$prefix}btn-active-border-color: #{$black};
+  --#{$prefix}btn-disabled-color: #{$gray-500};
+  --#{$prefix}btn-disabled-bg: transparent;
```

Comment:
Why not `btn-disabled-bg-color` instead of `btn-disabled-bg`?

Reply:
Same as above.